### PR TITLE
App

### PR DESCRIPTION
**Pull request by jonathastavares**
In this milestone I created a simple Todo app with the following features:
- Add item function
- Delete item function
- Edit item function
- Homepage
- About page

Built with:
- React
- JavaScript
- HTML
- CSS

### CODE REVIEW

**Comment by ahmedadel56** [comment]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

Capture


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/functionBased/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/button-has-type */
```
Comment (ahmedadel56) [inline src/functionBased/components/Navbar.js]: Please Remove that line and try to solve the linter error

File: src/functionBased/components/TodoContainer.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-use-before-define, no-param-reassign */
```
Comment (ahmedadel56) [inline src/functionBased/components/TodoContainer.js]: As here too, you should fix the linter error instead of disabling it

File: README.md [hunk 3]
```diff
@@ -1,2 +1,53 @@
...
+
+4 - start your local server and build project using
+
+`npm start`
+
+## Authors
+
+👤 **Jonathas Tavares**
+
+- GitHub: [jonathastavares](https://github.com/jonathastavares)
```
Comment (ahmedadel56) [inline README.md]: please try to pass the readme file to Grammarly to check grammar problems

**Review by ahmedadel56 — CHANGES_REQUESTED** [review]

**Comment by ShadyShawkat** [comment]
# Hi @jonathastavares, 👋🏻  

You have done awesome work so far, and it shows! 🔥 🚀 
I don't have any major change for you as of right now.

## Optional suggestions

- You can add a live demo link in your README.md file, which makes it easier to access your project.
- An image in your README.md file would be awesome to better showcase the look of your project.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**